File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/ItemList/Paginator.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Paginator {

    @SerializedName("cur_page")
    @Expose
    private String curPage;
    @SerializedName("num_pages")
    @Expose
    private int numPages;

    public String getCurPage() {
        return curPage;
    }

    public void setCurPage(String curPage) {
        this.curPage = curPage;
    }

    public int getNumPages() {
        return numPages;
    }

    public void setNumPages(int numPages) {
        this.numPages = numPages;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itemlist/Paginator.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Paginator {

    @SerializedName("cur_page")
    @Expose
    var curPage: String? = null
    @SerializedName("num_pages")
    @Expose
    var numPages: Int = 0

}
